meta-iot2000-bsp/recipes-devtools/qt5/iot2000-example-image.bbappend

```diff
@@ -0,0 +1 @@
+inherit populate_sdk_qt5
```

Comment:
Style: please add a new line at the end here.

Does this populate the SDK with QT bits even if the user does not include QT? My first guess would be "yes", and that would not be desirable.

Reply:
Sorry, I didn't realize could lead to that. Maybe we can put `INHERIT += " populate_sdk_qt5"`
somewhere, in `qt5-iot2000.inc` perhaps?! But I think it only works inside configuration files.
This is only needed in order to export qt5 libraries during sdk creation.